ui/admin/app/components/auth-methods/auth-method/actions/index.hbs

```diff
@@ -4,78 +4,64 @@
 }}
 
 {{#if (or @model.isOIDC @model.isLDAP)}}
-  <Rose::Dropdown
-    @text={{t (concat 'states.' @model.state)}}
-    @icon={{if
-      @model.isInactive
-      'flight-icons/svg/x-circle-16'
-      (if
-        @model.isPrivate
-        'flight-icons/svg/lock-16'
-        'flight-icons/svg/check-circle-16'
-      )
-    }}
-    @dropdownRight={{true}}
-    as |dropdown|
-  >
-    <dropdown.section @title={{t 'actions.change-state'}}>
-      <Rose::Form as |form|>
-        <form.radioGroup
-          @name='theme'
-          @selectedValue={{@model.state}}
-          @changed={{fn @changeState @model}}
-          as |radioGroup|
-        >
-          {{#each @model.options.state as |state|}}
-            <dropdown.item>
-              <radioGroup.radio
-                @label={{t (concat 'states.' state)}}
-                @value={{state}}
-              />
-            </dropdown.item>
-          {{/each}}
-        </form.radioGroup>
-      </Rose::Form>
-    </dropdown.section>
-  </Rose::Dropdown>
+  <Hds::Dropdown class='change-state' data-test-change-state as |dd|>
+    <dd.ToggleButton
+      @text={{t (concat 'states.' @model.state)}}
+      @color='secondary'
+      @icon={{if
+        @model.isInactive
+        'x-circle'
+        (if @model.isPrivate 'lock' 'check-circle')
+      }}
+    />
+
+    {{#each @model.options.state as |state|}}
+      <dd.Radio
+        checked={{eq @model.state state}}
+        @value={{state}}
+        {{on 'click' (fn @changeState @model state)}}
+      >
+        {{t (concat 'states.' state)}}
+      </dd.Radio>
+    {{/each}}
+  </Hds::Dropdown>
+
 {{/if}}
 
 <Hds::Dropdown data-test-manage-auth-method as |dd|>
   <dd.ToggleButton @text={{t 'actions.manage'}} @color='secondary' />
   {{#if @model.isPrimary}}
-    <dd.Interactive
-      @text={{t 'resources.auth-method.actions.remove-as-primary'}}
-      {{on 'click' (fn @removeAsPrimary @model)}}
-    />
+    <dd.Interactive {{on 'click' (fn @removeAsPrimary @model)}}>
+      {{t 'resources.auth-method.actions.remove-as-primary'}}
```

Comment:
this is the new format, instead of text attr, we need to yield it, I updated these as I was making changes in the same change. We have a separate [ticket](https://hashicorp.atlassian.net/browse/ICU-15713) to update it in other places